commands/level/level_rankcard.py

```
@@ -1,20 +1,24 @@
 from typing import Any
 
-import aiohttp
 import discord
-from aiohttp import ClientTimeout
-from PIL import Image, ImageDraw, ImageFont, ImageSequence
+from PIL import Image, ImageDraw
 
 from api import get_user_level_info, set_custom_background
 from localizer import tanjunLocalizer
 from models import UserLevelInfoModel
+from services.pillow_service import (
```

Flake8 (7.3.0)

[error] 'services.pillow_service.fetch_image' imported but unused

(F401)